pydis_site/apps/api/viewsets/bot/nomination.py

```diff
@@ -148,10 +160,44 @@ class NominationViewSet(CreateModelMixin, RetrieveModelMixin, ListModelMixin, Ge
     serializer_class = NominationSerializer
     queryset = Nomination.objects.all()
     filter_backends = (DjangoFilterBackend, SearchFilter, OrderingFilter)
-    filter_fields = ('user__id', 'actor__id', 'active')
-    frozen_fields = ('id', 'actor', 'inserted_at', 'user', 'ended_at')
+    filter_fields = ('user__id', 'active')
+    frozen_fields = ('id', 'inserted_at', 'user', 'ended_at')
     frozen_on_create = ('ended_at', 'end_reason', 'active', 'inserted_at')
 
+    def list(self, request: HttpRequest, *args, **kwargs) -> Response:
+        """
+        DRF method for listing Nominations.
+
+        Called by the Django Rest Framework in response to the corresponding HTTP request.
+        """
+        queryset = self.filter_queryset(self.get_queryset())
+        data = NominationSerializer(queryset, many=True).data
+
+        for i, nomination in enumerate(data):
+            entries = NominationEntrySerializer(
+                NominationEntry.objects.filter(nomination_id=nomination["id"]),
+                many=True
+            ).data
+            data[i]["entries"] = entries
```

Comment:
Wouldn't this be solved by adding a `serializers.PrimaryKeyRelatedField` to the `NominationSerializer`? That saves us from doing a manual `for`-loop here. In fact, it probably means we wouldn't have to overwrite the list method at all, as we don't have need for custom functionality that way.